src/System.IO.Pipelines/tests/BackpressureTests.cs

```diff
@@ -38,7 +38,9 @@ public void AdvanceThrowsIfFlushActiveAndNotConsumedPastThreshold()
 
             ReadResult result = _pipe.Reader.ReadAsync().GetAwaiter().GetResult();
             SequencePosition consumed = result.Buffer.GetPosition(31);
-            Assert.Throws<InvalidOperationException>(() => _pipe.Reader.AdvanceTo(consumed, result.Buffer.End));
+
+            var exception = Assert.Throws<InvalidOperationException>(() => _pipe.Reader.AdvanceTo(consumed, result.Buffer.End));
+            Assert.Equal("The maximum buffer limit of 32 bytes was exceeded by the writer.", exception.Message);
```

Comment:
We generally avoid doing this; it'll fail if we ever localize.

Reply:
Fix it when you localize? 😄

Reply:
Oh the debt we'd have.